parity/export_hardcoded_sync.rs

```diff
@@ -96,7 +96,7 @@ pub fn execute(cmd: ExportHsyncCmd) -> Result<String, String> {
 	let hs = service.client().read_hardcoded_sync()
 		.map_err(|e| format!("Error reading hardcoded sync: {}", e))?;
 	if let Some(hs) = hs {
-		Ok(::serde_json::to_string_pretty(&hs.to_json()).expect("generated JSON is always valid"))
+		Ok(format!("{:#?}", hs))
```

Comment:
printing debug is as readable and pretty as printing json in this case

Reply:
I guess Json is used to embed `hardcodedSync` easily into a chain spec.

Reply:
(so I would rather keep the previous behavior)

Reply:
then it's as easy as it was before,

before: 

```
{
  "header": "0x...",
  "total_difficulty": "0x...",
  "chts": [
    "0x...",
  ]
}
```

now
```
"HardcodedSync": {
  "header": 0x...,
  "total_difficulty": 0x...,
  "chts": [
    0x...,
  ]
}
```

Reply:
notice the quotes around the hashes

Reply:
fixed